src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -35,6 +35,7 @@ export interface IOperator {
   selected?: boolean,
   dappNode?: boolean,
   ownerAddress: string,
+  dkg_address?: string,
```

Comment:
Can it be camel-case? 
The same question about verified_operator field

Reply:
this fields came from api as underscore